plugins/net_plugin/net_plugin.cpp

```diff
@@ -1410,12 +1421,21 @@ namespace eosio {
       enqueue_buffer( send_buffer, close_after_send );
    }
 
+   // called from connection strand
    void connection::enqueue_block( const signed_block_ptr& b, bool to_sync_queue) {
       peer_dlog( this, "enqueue block ${num}", ("num", b->block_num()) );
       verify_strand_in_this_thread( strand, __func__, __LINE__ );
 
       block_buffer_factory buff_factory;
       auto sb = buff_factory.get_send_buffer( b );
+      if( !sb ) {
```

Comment:
Remove this `if`, incomplete block is not possible for Mandel. Incomplete blocks were only possible for pruned blocks int `eosio/eos 2.1.x`

Reply:
Thanks. Removed the `if` statement.